acl_setup.php

```diff
@@ -174,6 +181,21 @@
  $gacl->add_object('patients', 'Lab Results (write,addonly optional)','lab', 10, 0, 'ACO');
      // xl('Lab Results (write,addonly optional)')
 
+
+ $gacl->add_object('groups', 'Add/Update groups','gadd', 10, 0, 'ACO');
+ // xl('Add/Update groups')
+ $gacl->add_object('groups', 'Create/Update groups appointment in calender','gcalendar', 10, 0, 'ACO');
+// xl('Create/Update groups appointment in calender')
+ $gacl->add_object('groups', 'Group log','glog', 10, 0, 'ACO');
+// xl('Group log')
+ $gacl->add_object('groups', 'Group detailed log of appointment in patient record','gdlog', 10, 0, 'ACO');
+// xl('Group detailed log of appointment in patient record')
+ $gacl->add_object('groups', 'Send message from the permanent group therapist to the personal therapist','gm', 10, 0, 'ACO');
```

Comment:
Revert the titles you changed in the following commit:
https://github.com/openemr/openemr/pull/598/commits/0fd37f3af29a8fa9304be4e276dc8ee65090fec4
These title were much more clear. And remember to spell calendar. And then also use these title in the below acl upgrade script.